### ISSUE TITLE
sticky-wrapper height set to zero on display:none change

### ISSUE THREAD

**Issue by garyhoffmann** [imakewebthings/waypoints#371]
I have a sticky top navigation.  When the screen resizes smaller, I need to remove this element and replace the navigation with a mobile style hamburger menu, so I set the display attribute of sticky-wrapper to none in CSS using media queries.  If I don't do this and the element is "stuck" when the screen resizes, the space is still consumed on the screen effectively leaving a gap on the screen if I only set the internal elements to display:none.  If this element is not yet stuck, then the height is set to zero upon this change.  The height attribute is not cleared or reset to some valid height when the display:none is removed (by sizing the window larger) effectively having the div display, but not taking up any real space on the page because the element still has style="height: 0".

How can I capture this and clear the height attribute properly?

This is an requirement for a responsive site going between landscape and portrait on tablets.  When in landscape, I'd like the regular menu, when in portrait I want the mobile menu.

**Comment by garyhoffmann** [comment]
As a side-note, I even tried wrapping an outer wrapper that comes before the sticky-wrapper and set this one to display:none and I have the same results

**Comment by imakewebthings** [comment]
I'm not sure this qualifies as a bug report according to the [Contributing Guidelines](https://github.com/imakewebthings/waypoints/blob/master/CONTRIBUTING.md). It may serve the community better as a question on Stack Overflow. That said, have you considered instead of messing with CSS during the resizes across responsive breakpoints, instead calling [sticky.destroy()](http://imakewebthings.com/waypoints/shortcuts/sticky-elements/#sticky-destroy) when resizing down and reinitializing the sticky when resizing up?

**Comment by garyhoffmann** [comment]
Hi Caleb - I apologize if this is not actually a bug - it seemed like one to me.  Anyway, after spending about 6 hours trying everything I was able to find on the web, I solved this a different way by putting the hamburger menu into the sticky wrapper and wrapping both navigations with a sticky-element div.  This way, when I turn off the main horizontal nav and turn on the mobile nav, both are happening inside a sticky element and the sizing, etc. still works OK, and in fact better as the mobile nav stays stuck to the top of the browser window which is better for mobile users.

I had not thought of the sticky.destory() - that may have worked as well, but since the site is now live, I'll leave good enough alone.
